<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.2">
    <title>Links for django-fallback-storage</title>
  </head>
  <body>
    <h1>Links for django-fallback-storage</h1>
<a href="https://files.pythonhosted.org/packages/db/0f/87fc8cd7a8187a845d57aa0c9ada3f5d511a32787415cf4d4710dbcab555/django-fallback-storage-1.0.0.tar.gz#sha256=ce61769dc6986d978e73c529510d7d1e719ad87f53392f3d6de0c1cf43423e85" >django-fallback-storage-1.0.0.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/18/c0/5d47ed30822fe6ee3be5adf35e90aff9e9efa22790634943455de3b37d0f/django_fallback_storage-1.0.0-py2.py3-none-any.whl#sha256=645ad6390a63e84f2b05a5f4a430b46a1924da3d5d23767270b9f581750888ab" data-dist-info-metadata="sha256=04a26ba5ebd22b944a315fb06e6c3a8122f9c88d83e9628b405611af56d36182" data-core-metadata="sha256=04a26ba5ebd22b944a315fb06e6c3a8122f9c88d83e9628b405611af56d36182">django_fallback_storage-1.0.0-py2.py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/f4/64/e0c0fdf6b8fd9891c32faaa399c5a2ebba35c0fa1c15cbe630c623069f00/django-fallback-storage-1.1.0.tar.gz#sha256=1d83416410aeadb760c0849ca4e9db7e0d1725b7a4ef8bad08bd91c8b8151649" >django-fallback-storage-1.1.0.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/91/39/a01cff6bc221a962e79e750ca6a91d2cf736583beb46a265f8feae067e5d/django_fallback_storage-1.1.0-py2.py3-none-any.whl#sha256=464a6dfa5f5e8cc69aa053647700a49229226159653197ddc024dc5b5a9a367f" data-dist-info-metadata="sha256=cdeab566a8a348fcbf7a28a18ba80e651b2f97ae4221f1c3e22ed9254c937bc0" data-core-metadata="sha256=cdeab566a8a348fcbf7a28a18ba80e651b2f97ae4221f1c3e22ed9254c937bc0">django_fallback_storage-1.1.0-py2.py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/53/34/1875fcafd5ead0033ab89ddc8ff680f27fa04aca27293cf94d7c69b8a678/django-fallback-storage-1.1.1.tar.gz#sha256=93571777e902f94e6fabbff8cfef5ac85850377e945bb32ed1a722722687626b" >django-fallback-storage-1.1.1.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/f8/67/918c7c8144e9c8c7d62821f3a0eb4e0fa162d9c5a4981ee868ac2c8e7f94/django_fallback_storage-1.1.1-py2.py3-none-any.whl#sha256=85fe7653857ed94745e98c941863322caa2dd8b662b83f69a35f7a25940f25ab" data-dist-info-metadata="sha256=b55b85e2e10ada71c2e0a7c146f26ba305db3a224e1d6222c347dff46c360d8d" data-core-metadata="sha256=b55b85e2e10ada71c2e0a7c146f26ba305db3a224e1d6222c347dff46c360d8d">django_fallback_storage-1.1.1-py2.py3-none-any.whl</a><br />
</body>
</html>
<!--SERIAL 3435224-->